### DISCUSSION TITLE
Guide to documentation

### DISCUSSION THREAD

**Discussion by JTSvejda** [thliebig/openEMS-Project#95] [General]
Hello,

I was wondering if there is already a guide on how to contribute to the new read-the-docs documentation (https://docs.openems.de/)?

Best,
Taro

**Comment by biergaizi** [comment]
I have a Pull Request on documentation improvement at https://github.com/thliebig/openEMS-Project/pull/77/files. But @thliebig said he's so far too busy to review them until the end of summer. Perhaps we can collaborate on that, with you working on top of the changes I already made.

**Reply by JTSvejda** [reply]
Sounds like a plan. I'll have a look into it and contact you again.

**Reply by thliebig** [reply]
I will gets the doc improvements merged as soon as I can... I really appreciate any help...
